SSZipArchive/SSZipArchive.m

```diff
@@ -393,6 +393,12 @@ + (BOOL)unzipFileAtPath:(NSString *)path
                 strPath = [strPath stringByReplacingOccurrencesOfString:@"\\" withString:@"/"];
             }
             
+            // Sanitize path traversal characters if they're present in the file name to prevent directory backtracking. Ignoring these characters mimicks the default behavior of the Unarchiving tool on macOS.
+            if ([strPath rangeOfString:@"../"].location != NSNotFound) {
```

Comment:
@EthanArbuckle , @jhudsonWA , If filename is exactly "." or "..", don't we risk to rewrite the permissions/date for destination folder or its parent?